src/components/JobCard/JobCard.tsx

```diff
@@ -0,0 +1,88 @@
+import { Grid, Avatar, Typography, Button, Chip, Card, CardContent } from '@mui/material'
+import BusinessIcon from '@mui/icons-material/Business'
+import LocationOnIcon from '@mui/icons-material/LocationOn'
+import AccountBalanceWalletIcon from '@mui/icons-material/AccountBalanceWallet'
+import AccessTimeIcon from '@mui/icons-material/AccessTime'
+import JobData from '@/types/JobListing'
+
+
+interface JobCardProps {
+    job: JobData
+}
+
+const JobCard: FC<JobCardProps> = ({ job }) => {
+    const handleReadMoreClick = () => {
+        window.open(job.jobPostingUrl, '_blank', 'noopener,noreferrer')
+    }
+
+    return <Card sx={{ bgcolor: 'primary.dark', width: '100%' }}>
+        <CardContent>
+            <Grid item xs={12} md={2} display={{ xs: 'flex', sm: 'flex', md: 'none' }} alignItems="center" justifyContent="right"  sx={{ marginBottom: { xs: 4, sm: 4 }, marginRight: 2 }}>
+                <AccessTimeIcon sx={{ marginRight: 1 }} />
+                <Typography fontSize="body2">
+                    {job.publicationDate}
+                </Typography>
+            </Grid>
+            <Grid container alignItems="center">
+                <Grid item xs={12} sm={12} md={2} display="flex" justifyContent="center">
+                    <Avatar
+                        sx={{ bgcolor: 'white', width: 135, height: 135, borderRadius: 2,  img: {objectFit: 'contain', width: '90%', height: '100%'}
+                        }}
+                        alt={`${job.company} logo`}
+                        src={job.logoUrl}
+                    />
+                </Grid>
+                <Grid item xs={12} sm={12} md={8} mt={2}>
+                    <Typography variant="h5" mb={2} mt={2} component="div" sx={{ textAlign: 'left' }}>
+                        {job.title}
+                    </Typography>
+                    <Grid container spacing={2}>
+                        <Grid item xs={12} sm={4} sx={{ display: 'flex', justifyContent: { xs: 'flex-start', sm: 'center' }, alignItems: 'center' }}>
+                            <BusinessIcon />
+                            <Typography fontSize="body2" sx={{ ml: 1 }}>{job.company}</Typography>
+                        </Grid>
+                        <Grid item xs={12} sm={4} sx={{ display: 'flex', justifyContent: { xs: 'flex-start', sm: 'center' }, alignItems: 'center' }}>
+                            <LocationOnIcon />
+                            <Typography fontSize="body2" sx={{ ml: 1 }}>{job.location}</Typography>
+                        </Grid>
+                        {job.salary != '' &&
+                            <Grid item xs={12} sm={4} sx={{ display: 'flex', justifyContent: { xs: 'flex-start', sm: 'center' }, alignItems: 'center' }}>
+                                <AccountBalanceWalletIcon />
+                                <Typography fontSize="body2" sx={{ ml: 1 }}>{job.salary}</Typography>
+                            </Grid>
+                        }
+                    </Grid>
```

Comment:
Let's align with the left of the container. Mixing `center` justification within left justification is visually confusing.

```suggestion
                    <Grid container spacing={2}>
                        <Grid item xs={12} sm={4} sx={{ display: 'flex', justifyContent:  'flex-start', alignItems: 'center' }}>
                            <BusinessIcon />
                            <Typography fontSize="body2" sx={{ ml: 1 }}>{job.company}</Typography>
                        </Grid>
                        <Grid item xs={12} sm={4} sx={{ display: 'flex', justifyContent: 'flex-start', alignItems: 'center' }}>
                            <LocationOnIcon />
                            <Typography fontSize="body2" sx={{ ml: 1 }}>{job.location}</Typography>
                        </Grid>
                        {job.salary != '' &&
                            <Grid item xs={12} sm={4} sx={{ display: 'flex', justifyContent: 'flex-start', alignItems: 'center' }}>
                                <AccountBalanceWalletIcon />
                                <Typography fontSize="body2" sx={{ ml: 1 }}>{job.salary}</Typography>
                            </Grid>
                        }
                    </Grid>
```

Reply:
<img width="1134" alt="Screenshot 2024-09-01 at 1 43 44 PM" src="https://github.com/user-attachments/assets/51a5f085-b41d-4e19-bbfb-7872ed9a90fb">